pkgs/sdk_triage_bot/lib/src/gemini.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:google_generative_ai/google_generative_ai.dart';
+import 'package:http/http.dart' as http;
+
+abstract class GeminiService {
+  Future<String> summarize(String prompt);
+  Future<String> classify(String prompt);
```

Comment:
A priori, I would expect classify to return a typed value - something more like `Future<Value> classify(String prompt, List<Value> categories)`, or even `Future<Value> classify(String prompt, Value Function(String) categoryMatcher)`. But I am fine with the user handling the part of matching the return value to the category as well.